🐛 Input validation bug fix

yomeshgupta opened the pull request:
Fixes #11049 

**Problem**: Input type number badInput validation will show valueMissing error message in Firefox and Chrome.

**Solution**: In case of additional validation error along with **valueMissing**, additional error is given precedence.

yomeshgupta commented:
CLA signed with both author ids.

YomeshAdPushup commented:
I signed it!

yomeshgupta commented:
I signed it!

aghassemi commented on extensions/amp-form/0.1/form-validators.js:
```diff
@@ -323,18 +305,16 @@ export class AbstractCustomValidator extends FormValidator {
       if (!inputs[i].checkValidity()) {
         this.reportInput(inputs[i]);
-        inputs[i]./*REVIEW*/focus();
+        inputs[i]./*REVIEW*/ focus();
```
@yomeshgupta Could you please revert the unrelated formatting changes? Maybe your IDE automatically added them.

aghassemi commented:
@yomeshgupta Thanks for the PR! there are commits from two different emails here (your personal and work ones). You would need to either sign the CLA with both emails or send another PR with commits from only the email that has signed the CLA.

yomeshgupta commented:
@aghassemi By mistake, commits were pushed from two different accounts. To resolve that, I have signed CLA from both my accounts.

I did run gulp lint --fix to take care of formatting changes. Is there some specific formatting required for the AMP project?

aghassemi commented:
@yomeshgupta We follow https://google.github.io/styleguide/jsguide.html for style and it is enforced by our Lint file.

Regarding CLA, you may need to close this PR and send another one where all commits are from the same email, unfortunately the CLA check bot will not allow merge until it is happy with the CLA

yomeshgupta commented:
@aghassemi Okay. Thank you for the letting me know about the style guide. 

I am closing this PR and creating a new PR in which I have reverted the formatting changes and fixed dual account issue and input validation issue. 

New PR - https://github.com/ampproject/amphtml/pull/18069
